interactions/api/models/message.py

```diff
@@ -790,6 +790,19 @@ async def delete(
             guild_id=guild_id, emoji_id=int(self.id), reason=reason
         )
 
+    @property
+    def url(self) -> str:
+        """
+        Returns the emoji's url.
+        :return URL of the emoji
+        :rtype: str
+        """
+        if not self._client:
+            raise AttributeError("HTTPClient not found!")
+        url = f"https://cdn.discordapp.com/emojis/{self.id}"
```

Comment:
```suggestion
        url = f"https://cdn.discordapp.com/emojis/{str(self.id)}"
```

Reply:
This _should_ be unneeded - f-strings should do that for you, and sourcery will complain too.

Reply:
It has a `__str__`, which Python will automatically use here.

Reply:
Totally forgot about that xD